src/noos_inv/python.py

```diff
@@ -103,6 +104,8 @@ def package(ctx: Context, install: str | None = None) -> None:
         ctx.run("poetry build", pty=True)
     if install_type == InstallType.pipenv:
         ctx.run("pipenv run python -m build -n", pty=True)
+    if install_type == InstallType.uv:
+        ctx.run("uvx --from build pyproject-build --installer uv")
```

Comment:
**Suggestion:** Add error handling for the new `uv` install type to ensure that any errors during the build or release process are caught and handled appropriately. [robustness, importance: 8]
```suggestion
    if install_type == InstallType.uv:
        try:
            ctx.run("uvx --from build pyproject-build --installer uv")
        except Exception as e:
            ctx.log(f"Error during UV build process: {str(e)}")
            raise
```